iam/get.go

```diff
@@ -101,9 +101,9 @@ func ListUsers(showOnly string) error {
 			for _, key := range user.accessKeys {
 				switch aws.StringValue(key.status) {
 				case "Active":
-					summaryStats["activeKeys"] += 1
+					summaryStats["activeKeys"]++
 				case "Inactive":
-					summaryStats["inactiveKeys"] += 1
+					summaryStats["inactiveKeys"]++
```

Comment:
![eyes bleed](https://media.giphy.com/media/lqVVqkqMolh9S/giphy.gif)

Reply:
I know...... I know. It's something. The go community prefers this.